cli/command/container/opts.go

```diff
@@ -899,3 +921,12 @@ func validateAttach(val string) (string, error) {
 	}
 	return val, errors.Errorf("valid streams are STDIN, STDOUT and STDERR")
 }
+
+func convertToNetworkmode(netmode opts.NetworkOpt) container.NetworkMode {
+	networkIDOrName := "default"
+	for _, net := range netmode.Value() {
+		networkIDOrName = net.Target
+		break
```

Comment:
docker run does not support multiple networks . if user does give multiple networks only the first network is considered.
So in the above code I decided to do the same as well to keep it in sync. 
And if no network options is given then the ID would be "default"